include/planar_sssp/PlanarSSSP.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <boost/functional/hash.hpp>
+#include <networkit/base/Algorithm.hpp>
+#include <networkit/graph/Graph.hpp>
+#include <optional>
+#include <set>
+
+namespace Koala {
+class PlanarSSSP : public NetworKit::Algorithm {
+   public:
+    virtual void run() = 0;
+
+    NetworKit::count getSSSDistanceToTarget();
```

Comment:
Please align how the results are returned with NetworKit::Dijkstra, that is, with [NetworKit::SSSP](https://networkit.github.io/dev-docs/cpp_api/classNetworKit_1_1SSSP.html#_CPPv4N9NetworKit4SSSPE)